pkg/services/alerting/engine.go

```diff
@@ -37,7 +37,7 @@ type AlertEngine struct {
 	Cfg              *setting.Cfg
 
 	execQueue          chan *Job
-	ticker             *Ticker
+	ticker             *ticker.Ticker
```

Comment:
The stdlib commonly does something like `ticker.T` in these cases to avoid repeating type names. Is there a general Grafana `time` package we could use? If so I think grouping it with other time utilities would be valuable.

Reply:
I like `ticker.T`

Reply:
After searching for a while I found no suitable existing package that this could be merged into.

Renamed `Ticker` to `T`, and also renamed `ticker.NewTicker` to `ticker.New` to match.

Reply:
`ticker.New` is super nice 👍